src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+
+set BridgeHost=
+cd %2
+REM ensure bridge is stopped
+call stopbridge.cmd
+REM remove bin directory to ensure a rebuild
+del /s /q bin
+del *.rsp
+
+REM sync to the latest
+git checkout master
+git pull
+
+REM start the bridge
+call startbridge.cmd  /allowRemote /remoteAddresses:*
+
+copy /y %1\*.rsp .
```

Comment:
The template comes from VSO private Repro. I keep our own ones in both Server machine and my own machine, not shared. The plan is to copy them to the same location of %1 on the server machine.